miner/src/main/java/fr/raksrinana/channelpointsminer/miner/prediction/bet/outcome/MostTrustedPicker.java

```diff
@@ -0,0 +1,86 @@
+package fr.raksrinana.channelpointsminer.miner.prediction.bet.outcome;
+
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import fr.raksrinana.channelpointsminer.miner.api.ws.data.message.subtype.Outcome;
+import fr.raksrinana.channelpointsminer.miner.database.IDatabase;
+import fr.raksrinana.channelpointsminer.miner.database.NoOpDatabase;
+import fr.raksrinana.channelpointsminer.miner.database.model.prediction.OutcomeStatistic;
+import fr.raksrinana.channelpointsminer.miner.handler.data.BettingPrediction;
+import fr.raksrinana.channelpointsminer.miner.prediction.bet.exception.BetPlacementException;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import lombok.extern.log4j.Log4j2;
+import org.jetbrains.annotations.NotNull;
+import java.util.Comparator;
+
+@JsonTypeName("mostTrusted")
+@Getter
+@EqualsAndHashCode
+@ToString
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Log4j2
+public class MostTrustedPicker implements IOutcomePicker{
+	
+	@Builder.Default
+	private int minTotalBetsPlacedByUser = 5;
+	@Builder.Default
+	private int minTotalBetsPlacedOnPrediction = 10;
+	@Builder.Default
+	private int minTotalBetsPlacedOnOutcome = 5;
+	
+	@Override
+	@NotNull
+	public Outcome chooseOutcome(@NotNull BettingPrediction bettingPrediction, @NotNull IDatabase database) throws BetPlacementException{
+		
+		try{
+			if(database instanceof NoOpDatabase){
+				throw new BetPlacementException("A database needs to be configured for this outcome picker to work");
+			}
+			
+			var outcomes = bettingPrediction.getEvent().getOutcomes();
+			var title = bettingPrediction.getEvent().getTitle();
+			
+			var outcomeStatistics = database.getOutcomeStatisticsForChannel(bettingPrediction.getEvent().getChannelId(), minTotalBetsPlacedByUser);
```

Comment:
I think it'd be nice to have a configuration to keep channels with only a winRate greater than what is defined.

For example if a channel has only 10 betters all the time but they always lose, I don't want to follow their choice.